lib/vanagon/platform/windows.rb

```diff
@@ -170,7 +162,7 @@ def output_dir(target_repo = "")
       #
       # @return [String] the cygwin drive root
       def drive_root
-        "/cygdrive/c"
+        "C:"
```

Comment:
Hmm, yeah, you're right. That would take more refactoring to make possible. But that is a very compelling point you make. I'm thinking we do that if someone opens a ticket or asks for it specifically. There are currently only two places in vanagon that reference `C:`. The build environment we're depending on is so strange, I have a feeling this will be the least of our problems when the public starts to take advantage of this. But, if you feel strongly about it, I can try to make it happen.